src/styles/blocks/placeholder.scss

```diff
@@ -0,0 +1,6 @@
+%font-500 {
```

Comment:
Rename file to `_extends.scss` and move to `utils` folder.
More about naming in this [video, timing 1:45](https://mate.academy/learn/css-advanced/sass#/video/931)